kdestandardize.ado

```diff
@@ -1399,18 +1412,51 @@ prog def kdestandardize
 									
 	} // End of handling of the Equity_Measure  variable
 	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
-	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
-	
-	
-	  
+	// Handles instances of the EQ_LABEL variable
+	if `: list posof "EQ_LABEL" in x' != 0 {
+		qui: replace EQ_LABEL=cond(`rx'(EQ_LABEL, "Exemplary/ Accomplished", 1), "1", ///
```

Comment:
No need to use a regular expression here if you're trying to do an exact match on a string.  It will require more execution time than the equality check and requires more code than necessary for this logic test.